website/docusaurus.config.js

```diff
@@ -29,14 +29,15 @@ const config = {
           '../packages/dev-utils',
           '../packages/plugin-cli',
         ],
+        tsconfig: '../packages/plugin-cli/tsconfig.json',
```

Comment:
Why do we need to specify tsconfig and why is it leading to one specific package? Isn't the root `tsconfig.json` file enough? If not, shouldn't we create a new `tsconfig` file inside `website` folder for this specific build?